.flake8

```diff
@@ -2,4 +2,11 @@
 import-order-style = google
 application-import-names = e2c, tests
 per-file-ignores =
-    tests/*: D104
+    # D104 Missing docstring in public package
+    tests/__init__.py: D104,
+    tests/e2c/__init__.py: D104,
```

Comment:
Why?

Reply:
I just removed that term and add some docstring in those two `__init__.py` files